api/src/main/java/au/org/aodn/nrmn/restapi/validation/validators/row/formatted/MeasureBetweenL5l95.java

```diff
@@ -28,11 +28,7 @@ public MeasureBetweenL5l95() {
         if (skipMethods.contains(target.getMethod())) {
             return Validated.valid("M3, M4, M5 species");
         }
-
-        if(!target.getRef().getStagedJob().getIsExtendedSize()) {
```

Comment:
Need to modify code below to handle target.getIsInvertSized now possibly being null.

Reply:
Or better ye - only refering to it if extendedSizing applies